code/pcgen.sh

```diff
@@ -82,5 +82,5 @@ done
 #     -Dpcgen.filter=/path/to/filter.ini
 #     -Dpcgen.options=/path/to/options.ini
 
-cd $(dirname $(readlink -f $0))
+# shellcheck disable=SC2086
```

Comment:
Why is this commented out? Doesn't follow the principles of clean code.

Reply:
This is a "lint supression". It is supposed to be a comment. It tells the tool 'shellcheck' not to report `$javaargs` as unquoted. we want it as such in order to expand.

Reply:
Gotcha.

Reply:
I should install a linux build to properly test these linux only issues...